ajax/libs/css-doodle/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "css-doodle",
+  "filename": "css-doodle.min.js",
+  "description": "A web component for drawing patterns with css.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/yuanchuan/css-doodle.git"
+  },
+  "keywords": [
+    "css",
+    "web-components"
+  ],
+  "author": "yuanchuan",
+  "license": "MIT",
+  "npmName": "css-doodle",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css-doodle*.js"
```

Comment:
Let's use `"css-doodle*"`
We can fetch CSS files if the lib provide the file in future.
Many thanks.

Reply:
Sorry for that. I've made the change. Please check it!

Thanks!